python/tank_vendor/shotgun_api3/lib/mockgun.py

```diff
@@ -347,13 +347,29 @@ def find(self, entity_type, filters, fields=None, order=None, filter_operator=No
             
         results = [row for row in self._db[entity_type].values() if self._row_matches_filters(entity_type, row, resolved_filters_2, filter_operator, retired_only)]
         
+        # handle the ordering of the recordset
+        if order:
+            for order_entry in order:
+                if "field_name" not in order_entry:
+                    raise ValueError("Order clauses must be list of dicts with keys field_name and direction!")
```

Comment:
same nitpicky comment here about distinguishing the key names with single quotes.